docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,502 @@
+    resort.
+
+* Run ``crate-node unsafe-bootstrap`` on the first remaining node
+  and answer ``n`` at the confirmation step.
 
 .. code-block:: console
 
     node_1$ ./bin/crate-node unsafe-bootstrap
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
     Current node cluster state (term, version) pair is (4, 12)
 
-    You should only run this tool if you have permanently lost half or more
-    of the master-eligible nodes in this cluster, and you cannot restore the
-    cluster from a snapshot. This tool can cause arbitrary data loss and its
-    use should be your last resort. If you have multiple surviving master
-    eligible nodes, you should run this tool on the node with the highest
-    cluster state (term, version) pair.
-
     Do you want to proceed?
 
-    Confirm [y/N] n
+    Confirm [y/n] n
 
-*   Run the tool on the second remaining node, and again answer n at the confirmation step.
+* Run ``crate-node unsafe-bootstrap`` on the second remaining node
+  and answer ``n`` at the confirmation step.
 
 .. code-block:: console
 
     node_2$ ./bin/crate-node unsafe-bootstrap
 
-       WARNING: CrateDB MUST be stopped before running this tool.
-
     Current node cluster state (term, version) pair is (5, 3)
 
-    You should only run this tool if you have permanently lost half or more
-    of the master-eligible nodes in this cluster, and you cannot restore the
-    cluster from a snapshot. This tool can cause arbitrary data loss and its
-    use should be your last resort. If you have multiple surviving master
-    eligible nodes, you should run this tool on the node with the highest
-    cluster state (term, version) pair.
-
     Do you want to proceed?
 
-    Confirm [y/N] n
+    Confirm [y/n] n
+
+Since the second node has a greater term it has a fresher cluster state, so it
+is better to unsafely bootstrap the cluster using the second node.
 
-* Since the second node has a greater term it has a fresher cluster state, so
-  it is better to unsafely bootstrap the cluster using this node:
+* Run ``crate-node unsafe-bootstrap`` on the second remaining node and answer
+  ``y`` at the confirmation step.
 
 .. code-block:: console
 
     node_2$ ./bin/crate-node unsafe-bootstrap
 
-      WARNING: CrateDB MUST be stopped before running this tool.
-
     Current node cluster state (term, version) pair is (5, 3)
 
-    You should only run this tool if you have permanently lost half or more
-    of the master-eligible nodes in this cluster, and you cannot restore the
-    cluster from a snapshot. This tool can cause arbitrary data loss and its
-    use should be your last resort. If you have multiple surviving master
-    eligible nodes, you should run this tool on the node with the highest
-    cluster state (term, version) pair.
-
     Do you want to proceed?
 
-    Confirm [y/N] y
-    Master node was successfully bootstrapped
+    Confirm [y/n] y
```

Comment:
N is uppercase on purpose, since it is the default option when you just press enter. This is common with command line tools. So it should be Confirm [y/N]